Sources/ApolloCodegenLib/Frontend/JavaScript/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./tsconfig.base",
   "compilerOptions": {
-    "types": [],
+    "types": ["node"],
```

Comment:
If I recall correctly this is for certain Node types to be recognized by the compiler correct?

Reply:
Yeah, I needed it in order for `console.log` to work. But that stuff shouldn't get compiled into the rolled up stuff, so we should be good there.